api/proto/certs/certs.proto

```diff
@@ -0,0 +1,48 @@
+// Copyright(c) 2020 Zededa, Inc.
+// All rights reserved.
+
+syntax = "proto3";
+
+option go_package  = "github.com/lf-edge/eve/api/go/certs";
+option java_package = "com.zededa.cloud.uservice.proto";
+
+// This is the request payload for POST /api/v1/edgeDevice/EdgeNodeCerts
+// or /api/v2/edgeDevice/EdgeNodeCerts
+// ZEdgeNodeCert carries a set of X.509 certificate and their properties
+// from EVE to Controller.
+// The message is assumed to be protected by a TLS session bound to the
+// device certificate for v1.
+// The message is assumed to be protected by signing envelope for v2.
+message ZEdgeNodeCert {
+  string deviceUuid = 1;      //UUID of the device, assigned during register
+  repeated ZCert certs = 2;   //list of certificates sent by edge node
+}
+
+// This is the response payload for GET /api/v1/edgeDevice/ControllerCerts
+// or /api/v2/edgeDevice/ControllerCerts
+// ZEdgeNodeCert carries a set of X.509 certificate and their properties
+// from Controller to EVE.
+message ZControllerCert {
+  repeated ZCert certs = 1;  //list of certificates sent by controller
+}
+
+message ZCert {
+   string certId = 1;              //Identifier for the cert, usually a short hash of the cert
+   ZCertType type = 2;             //what kind of certificate(to identify the target use case)
+   string cert = 3;                //X509 cert in .PEM format
```

Comment:
I think we use bytes for all the new places we put certs. Need to check. PEM is ASCII

Reply:
changed to bytes.